src/metabase/db/liquibase.clj

```diff
@@ -263,13 +266,23 @@
 (defn release-concurrent-locks!
   "Release any locks held by this process corresponding to the same database."
   [conn-or-data-source]
+  ;; Check whether there are Liquibase locks held by the current process - we don't want to release the database locks
+  ;; if they are held by another server, for example if this host is part of an "old" fleet shutting down while new
```

Comment:
```suggestion
  ;; Check whether there are Liquibase locks held by the current instance - we don't want to release the database locks
  ;; if they are held by another instance, for example if this host is part of an "old" fleet shutting down while new
```

IMO we often use "instance" in this context

Reply:
I worry this could be confusing as we're also talking about Liquibase and Lock Service "instances" in the same comment - also I don't want to rerun CI for such a minor tweak 😋